### ISSUE TITLE
Error when build with webpack/UglifyJs

### ISSUE THREAD

**Issue by iamdubx** [matheusgrieger/vue-clazy-load#1]
Hi. Thanks for great module!

I am running `npm run build` with vue-cli webpack template and got an error

```
ERROR in static/js/vendor.636071240149cb133e9f.js from UglifyJs
Unexpected token: punc (() [./~/vue-clazy-load/src/clazy-load.js:8,0][static/js/vendor.636071240149cb133e9f.js:33092,9]
```
Is it critical?

**Comment by matheusgrieger** [comment]
Hello and thanks for asking!

I solved it for 0.1.5, can you test again, please?

**Comment by iamdubx** [comment]
@matheusgrieger now everything perfect, thanks for updating!
